.changeset/real-onions-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro-og-canvas': minor
+---
+
+Add support for Astro 3.0 & remove support for Astro 1.0 and 2.0
```

Comment:
We should probably add a migration note here explaining the additional action needed after updating:

````suggestion
Add support for Astro 3.0 and remove support for Astro 1.0 and 2.0.

When upgrading, update your open-graph routes to use `GET` instead instead of lowercase `get`:

```diff
import { OGImageRoute } from 'astro-og-canvas';

- export const { getStaticPaths, get } = OGImageRoute({
+ export const { getStaticPaths, GET } = OGImageRoute({
  // ...
});
```
````